app_dart/lib/src/request_handlers/reset_prod_task.dart

```diff
@@ -44,7 +45,12 @@ class ResetProdTask extends ApiRequestHandler<Body> {
   static const String ownerParam = 'Owner';
   static const String repoParam = 'Repo';
   static const String commitShaParam = 'Commit';
-  static const String builderParam = 'Builder';
+
+  /// Name of the task to be retried.
+  ///
+  /// If "all" is given, all failed tasks will be retried. This enables
+  /// oncalls to quickly recover a commit without the tedium of the UI.
+  static const String taskParam = 'Task';
```

Comment:
I'll update the oncall playbooks once this lands to refer to this. The builder terminology should be deprecated in favor of task/target.

Reply:
As a side note, Casey mentioned that we should also supply the 'branch' parameter